docs/develop/contracts/js/quickstart.md

```diff
@@ -0,0 +1,328 @@
+mkdir src && cd src && touch index.js && cd ..
+```
+
+The last step is to create a new file called `babel.config.json` which allows you to configure how the contract is built. In the project root, create a new file and add the following content.
+
+```bash
+touch babel.config.json
+```
+```js
+{
+  "plugins": [
+    "near-sdk-js/src/build-tools/near-bindgen-exporter",
+    ["@babel/plugin-proposal-decorators", {"version": "legacy"}]
+  ]
+}
+```
+
+At this point, you just need to install all the packages and you will be ready to build your smart contract!
+
+```bash
+npm install
+```
+
+Your file structure should now look as follows:
+
+```
+javascript-enclave-quickstart
+├── node_modules
+├── package-lock.json
+├── babel.config.json
+├── package.json
+└── src
+    └── index.js
+```
+
+### Writing your first contract
+
+Now that you have the basic structure outlined for your project, it's time to start writing your first contract. You'll create a simple contract for setting and getting a greeting message on-chain.
+
+The contract presents 2 methods: ```set_greeting``` and ```get_greeting```. ```set_greeting```stores a String in the contract's parameter message, while ```get_greeting``` retrieves it. By default, the contract returns the message "Hello".
+
+Start by opening the `src/index.js` file as this is where your logic will go. You'll then want to add some imports that will help when writing the contract:
+
+```js
+import {NearContract, NearBindgen, call, view, near} from 'near-sdk-js'
+```
+Let's break down these imports to help you understand why they're necessary.
+- `NearContract`: allows our contract to inherit functionalities for changing and reading the contract's state. State can be thought of as the data stored on chain. 
+- `NearBindgen`: allows your contract to compile down to something that is NEAR compatible.
+- `call, view`: allows your methods to be view only functions or mutable (change) functions.
+- `near`: allows you to access important information within your functions such as the signer, predecessor, attached deposit etc..
+
+Now that you've imported everything from the sdk, create a new class that extends the `NearContract`. This class will contain the core logic of your smart contract. You can also use this opportunity to create a default message variable. Below the import add: 
+
+```js
+// Define the default message
+const DEFAULT_MESSAGE = "Hello";
+
+@NearBindgen
+class StatusMessage extends NearContract {
+    // Define the constructor, which initializes the contract with a default message
+    constructor() {
+        // Used to give access to methods and properties of the parent or sibling class
+        super()
+        // Default the status records to 
+        this.message = DEFAULT_MESSAGE
+    }
+}
+```
+Running the constructor will default the contract's `message` state variable with the `DEFAULT_MESSAGE`. There's no way to get the current greeting, however. Within the class, add the following function.
```

Comment:
```suggestion
Running the constructor will default the contract's `message` state variable with the `DEFAULT_MESSAGE`. Since there's no way to get the current greeting, within the class you can add the following `view` function:
```